pipeline/merger/src/test/scala/weco/pipeline/merger/rules/ImageDataRuleTest.scala

```diff
@@ -180,6 +180,30 @@ class ImageDataRuleTest
     }
   }
 
+  it(
+    "correctly identifies a digmiro Sierra work regardless of other digcodes"
```

Comment:
What does this mean? 😅

Reply:
A Sierra Work might be marked with digmiro to show that its Miro content has been superseded by a METS version.

The problem this fixes is that the application could only identify a digmiro work if the `digmiro` is the first `digcode` on that Work.

Reply:
This test is essentially a copy of the preceding one, but with some extra digcodes